docs/typelibrary/devices/BeagleBoneBlack.md

```
@@ -1 +1,33 @@
 ### BeagleBoneBlack
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)